test/ctia/test_helpers/aggregate.clj

```diff
@@ -37,17 +37,45 @@
   [values]
   (mapcat set values))
 
+(defn l-get-in
+  "like get-in, but match keys in maps embedded in collections.
+  ~~~clojure
+  > (l-get-in {:a [{:b 2} {:b 3}]} [:a :b])
```

Comment:
It'd sound desirable to replace most of the docstring with tests.

Tests-as-documentation can be a great thing to be emphasized, while large docstrings can be awkward in IDE widgets.

Reply:
that's already a helper for testing, it would be meta tests :-)?

Reply:
BTW I could add some assert?

Reply:
I'm all for meta ;p

The following works (`:test` has an official meaning in Clojure):

```
(defn foo
  {:test (fn []
           (is ...))}
  []
  )
```

CIDER, Lein, etc all will run them as any other deftest.

These inlined tests seem a nice middle ground between having real coverage, and some accessible doc.